substrate/frame/staking/src/pallet/mod.rs

```diff
@@ -1872,6 +1881,33 @@ pub mod pallet {
 			ensure_signed(origin)?;
 			Self::do_payout_stakers_by_page(validator_stash, era, page)
 		}
+
+		/// Migrates an account's `RewardDestination::Controller` to
+		/// `RewardDestination::Account(controller)`.
+		///
+		/// Effects will be felt instantly (as soon as this function is completed successfully).
+		///
+		/// This will waive the transaction fee if the `payee` is successfully migrated.
+		#[pallet::call_index(27)]
+		#[pallet::weight(T::WeightInfo::set_payee())]
```

Comment:
This should probably have its own benchmark because it does an additional storage read compared to `set_payee`.

Reply:
Yes its coming 👍